discord_connector.py

```diff
@@ -3,16 +3,19 @@
 # -----------------------------------------
 # Phantom sample App Connector python file
 # -----------------------------------------
+# from lib2to3.fixes.fix_input import context
 
 # Phantom App imports
 import phantom.app as phantom
 from phantom.action_result import ActionResult
 from phantom.base_connector import BaseConnector
 
 # Usage of the consts file is recommended
-# from discord2_consts import *
+# from discord_consts import *
```

Comment:
unused?

Reply:
It is used now and it is going to be used much more in the near future.